apps/www/middleware.ts

```diff
@@ -133,13 +133,20 @@ async function middlewareFunc(req: NextRequest): Promise<NextResponse> {
   }
 
   const resUrl = req.nextUrl.clone()
+
   // Block if request is coming from IP_BLOCKLIST
   const clientIp = req.headers['x-forwarded-for']
     ? req.headers['x-forwarded-for'].split(',')[0]
     : ''
 
-  if (clientIp === '129.80.232.132') {
-    console.log(`request with IP: ${clientIp} denied.`)
+  const isBlacklistedIP =
+    clientIp &&
+    process.env.IP_BLOCKLIST &&
+    (process.env.IP_BLOCKLIST || '')
+      .split(',')
+      .some((ip) => clientIp.includes(ip))
+
+  if (isBlacklistedIP) {
```

Comment:
do we need a more professional server response or we dgaf?

Reply:
DGAF!!! :D